apps/web/pages/_app.tsx

```diff
@@ -1,17 +1,32 @@
-import localFont from "@next/font/local";
+import "../styles/global.css";
+import Script from "next/script";
+import Head from "next/head";
 import { AppProps } from "next/app";
-import { ReactElement } from "react";
-import "../styles/global.scss";
+import { wrapper } from "../store/store";
 
-const interFont = localFont({ src: "../assets/Inter-Regular.ttf" });
-
-export default function MyApp({
-  Component,
-  pageProps,
-}: AppProps): ReactElement {
+function App({ Component, pageProps }: AppProps): JSX.Element {
   return (
-    <main className={interFont.className}>
+    <>
+      <Head>
+        <style>
+          @import
+          url("https://fonts.googleapis.com/css2?family=Saira:wght@100&family=Press+Start+2P&family=Rubik+Vinyl&family=Lacquer&display=swap");
+        </style>
```

Comment:
There is a better way to import Google fonts (https://nextjs.org/docs/basic-features/font-optimization#google-fonts). On a side note, I think the most optimized way is to download the font a serve it as a local one.

Reply:
ok i didn't know thanks ;)

Reply:
Not exactly true to use in other css modules